Dockerfile

```diff
@@ -42,5 +42,11 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 COPY hack/gencerts.sh /usr/bin/
 
+# Setup dependencies for AWS S3 access
```

Comment:
Can we create a separate copy named `Dockerfile.aws` and make the changes in that copy?

Reply:
@liyinan926 the idea was to have this support in the original Dockerfile so ppl do not need to build images by themselves. 

Also, a new Docker file may start to diverge from the original one as time flies and this is also unwanted effect.

Reply:
@liyinan926 can we proceed this way?